spec/support/request/web_helper.rb

```diff
@@ -106,7 +106,6 @@ def fill_in_tag(tag_name, selector = ".tags-input .tags input")
     expect(page).to have_selector selector
     find(:css, selector).click
     find(:css, selector).set "#{tag_name}\n"
-    find(:css, selector).send_keys :enter
```

Comment:
So now it won't addTag until the focus is lost. If the expectation is passing, what caused the focus to be lost? 
Or is the expectation not checking for addTag, just checking that it was typed in?